hack/make-rules/clean.sh

```diff
@@ -25,7 +25,6 @@ CLEAN_PATTERNS=(
   "_tmp"
   "doc_tmp"
   "((?!staging\/src\/k8s\.io\/apiextensions-apiserver\/pkg\/generated\/openapi).)*/zz_generated.openapi.go"
-  "test/e2e/generated/bindata.go"
```

Comment:
Is this necessary?  Having clean purge that isn't the end of the world as long as regen is correct.

Reply:
I think this is desirable. Having `make clean` remove a committed file is what I would describe as surprising.